public/js/SpriteSheet.js

```diff
@@ -38,15 +38,15 @@ export default class SpriteSheet {
             return buffer;
         });
 
-        this.tiles.set(name, buffers);
+        this.tiles.set(style, buffers);
     }
 
-    defineTile(name, x, y) {
-        this.define(name, x * this.width, y * this.height, this.width, this.height);
+    defineTile(style, x, y) {
+        this.define(style, x * this.width, y * this.height, this.width, this.height);
     }
 
-    draw(name, context, x, y, flip = false) {
-        const buffer = this.tiles.get(name)[flip ? 1 : 0];
+    draw(style, context, x, y, flip = false) {
+        const buffer = this.tiles.get(style)[flip ? 1 : 0];
         context.drawImage(buffer, x, y);
     }
 
```

Comment:
The variable name of this file should not be changed.

Reply:
Fixed.